modules/openapi-generator/src/main/resources/typescript-node/api-all.mustache

```diff
@@ -8,5 +8,9 @@ import { {{ classname }} } from './{{ classFilename }}';
 export * from './{{ classFilename }}Interface'
 {{/withInterfaces}}
 {{/apis}}
+import http = require('http');
+export class HttpError {
```

Comment:
shouldn't this extend the javascript `Error` class to get the stack trace functionality that you are looking for?
```suggestion
export class HttpError extends Error {
```
See https://javascript.info/custom-errors

Reply:
yeah.. 🤦‍♂